Source/RunActivity/Viewer3D/RollingStock/MSTSLocomotiveViewer.cs

```diff
@@ -2454,7 +2454,8 @@ public CabViewDigitalRenderer(Viewer viewer, MSTSLocomotive car, CVCDigital digi
             if (Control.ControlType == CABViewControlTypes.CLOCK)
                 Alignment = LabelAlignment.Center;
             Alignment = digital.Justification == 1 ? LabelAlignment.Center : digital.Justification == 2 ? LabelAlignment.Left : digital.Justification == 3 ? LabelAlignment.Right : Alignment;
-
+            // Used for 3D cabs
+            Alignment = digital.Justification == 4 ? LabelAlignment.Cab3DCenter : digital.Justification == 5 ? LabelAlignment.Cab3DLeft : digital.Justification == 6 ? LabelAlignment.Cab3DRight : Alignment;
```

Comment:
I'm sure you have a reason for using 4, 5, 6 here instead of 1, 2, 3 so please can you put that reason into a comment here (I would rather use 1, 2, 3)

But, whatever that reason, I definitely do not see why you need those three additional values for `Alignment`. Why not read 4, 5, 6 and store as `LabelAlignment.Center`, `LabelAlignment.Left`, `LabelAlignment.Right`?

Reply:
I can put in a comment the explanation that I have written here http://www.elvastower.com/forums/index.php?/topic/35498-justification-alignment-for-digitals-in-3dcabs/page__view__findpost__p__275959 : as of now existing 3D cabs have left-aligned digitals independently from the value of Justification. If I now respect the value of Justification,, existing 3D cabs will change the alignment of their digitals, which is not desirable. 
Your proposal does not solve the above problem. If you prefer, I can use a new field in the .cvf control, e.g. ORTS3DAlignment, which can have values 1, 2 or 3, and check that one in the 3D cab code.

Reply:
Are there many 3D cabs that will seriously break if you used the same 1, 2, 3 values?

We are adding a lot of confusion complexity with 4, 5, 6 here so we need to justify that it is absolutely necessary.

Reply:
James, I see that especially commercial content developers are offering 3D cabs. I suppose they aren't happy if we break compatibility.
As written in my alternate proposal at the end of my message above yours, inserting ORTS3DAligment in the .cvf file there is no more need for values 4, 5 and 6. ORTS3DAlignment would have values 1, 2 and 3, and within the code it would be used to assign a LabelAlignment to a new variable Alignment3D, which would be the one checked in the 3D cab code.